dashboard/config/linux/bits/chromeos-5.15.yml

```diff
@@ -4,8 +4,8 @@
 kernel:
  repo: https://chromium.googlesource.com/chromiumos/third_party/kernel
  # HEAD of chromeos-5.15 branch.
- tag: 2ee474844f9c81d30317b9ad7f26009556414504
+ tag: 7e17ed0aa6753ba346037b6aa68c7b2b8d2c4744
 
 shell:
- - chromeos/scripts/prepareconfig chromiumos-x86_64 ${BUILDDIR}/.config
+ - CHROMEOS_KERNEL_FAMILY=chromeos chromeos/scripts/prepareconfig chromiumos-x86_64 ${BUILDDIR}/.config
```

Comment:
Not sure which one we should use here, I see there are several of them available:
```
CROS_KERNEL_FAMILY_VALUES=(
	arcvm
	chromeos
	manatee
	termina
)
```

Reply:
ChromeOS can run other OSes as guests. 

`chromeos` is the host kernel; the rest are guest kernels, e.g., `arcvm` is the Android kernel running in `ARCVM` (`crosvm`).

Reply:
If unsure, go withs `chromeos` :)

Reply:
yes, it should be good to go with "chromeos" as the others are VM kernels